Monika After Story/game/definitions.rpy

```diff
@@ -5,6 +5,102 @@ define config.developer = False #This is the flag for Developer tools
 init 1 python:
     persistent.steam = "steamapps" in config.basedir.lower()
 
+### Overrides of core renpy things
+python early in mas_overrides:
+    from store import *
```

Comment:
What do you need to import in here? We should only import what we need, or namespace the import.

Reply:
We import the entire store so we can use its content in functions calls. If we imported just store, we'd have to put `store.` before passing in each argument. Splitting args and kwargs to add the `store.` bit within the method is not possible.
```renpy
m "[func(arg1, arg2)]"
# becomes
m "[func(store.arg1, store.arg2)]"
```
Tbh now I think that `mas_overrides` is useless.

Reply:
I agree, keep it in the global store if you need everything from it anyway.